datasets/covid19_tracking/state_screenshots/custom/download_screenshots.py

```diff
@@ -17,42 +17,55 @@
 import os
 import pathlib
 import typing
+from multiprocessing.pool import ThreadPool
 
 import requests
-from google.cloud import storage
 
 
-def main(csv_path: pathlib.Path, source_column: str, target_column: str):
-    gcs_client = storage.Client()
+def download_item(source_target: typing.Tuple[str, pathlib.Path]):
```

Comment:
It's a bit funny to take a tuple as an argument. I guess this is to support `imap_unordered`, which can only pass in a single argument? Probably worth adding a docstring or comment explaining this.

Reply:
Yeah, I wanted to avoid using tuples but unfortunately `imap_unordered(func, iterable)` behaves such that an iterable of `[(1,2), (3, 4)]` results in `[func(1,2), func(3,4)]`.

I've added a docstring in 27eca8d.